docs/en/fuse-sdk/FUSE-SDK-Dora-Quick-Start.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: global
+title: FUSE SDK with Distributed Cache Quick Start
+group: FUSE SDK
+priority: 3
+---
+
+* Table of Contents
+  {:toc}
+
+![POSIX LOCAL CACHE](https://github.com/Alluxio/alluxio/blob/dora/docs/img/posix-distributed-cache.png)
+
+If FUSE SDK can optional provide L1 cache (local node metadata/data cache) capability,
+Alluxio cluster can provide L2 cache (local/nearby cluster metadata/data cache).
```

Comment:
no context for L1 vs L2 cache, doesn't seem appropriate to use without better explanation. i would stick to the verbose explanation

Reply:
The overview.md is the comparison

Reply:
overview first
- FUSE SDK with local cache quick start
- FUSE SDK with Dora distributed cache quick start